static/app/components/nav/secondarySidebar.tsx

```diff
@@ -2,26 +2,44 @@ import styled from '@emotion/styled';
 
 import {SECONDARY_SIDEBAR_WIDTH} from 'sentry/components/nav/constants';
 import {useNavContext} from 'sentry/components/nav/context';
+import {
+  NavTourElement,
+  STACKED_NAVIGATION_TOUR_CONTENT,
+  StackedNavigationTour,
+  useStackedNavigationTour,
+} from 'sentry/components/nav/tour/tour';
 
 export function SecondarySidebar() {
   const {setSecondaryNavEl} = useNavContext();
+  const {currentStepId} = useStackedNavigationTour();
+  const stepId = currentStepId ?? StackedNavigationTour.ISSUES;
 
   return (
     <SecondarySidebarWrapper
-      ref={setSecondaryNavEl}
-      role="navigation"
-      aria-label="Secondary Navigation"
-    />
+      id={stepId}
+      description={STACKED_NAVIGATION_TOUR_CONTENT[stepId].description}
```

Comment:
The secondary sidebar stays highlighted for the entirety of the tour, but the tooltip text will change